.github/workflows/sub_essential_tests.yml

```diff
@@ -82,6 +83,14 @@ jobs:
           timeout: "300"
           interval: "1"
 
+      - name: Wait for the wallet server to start and get ready
```

Comment:
We should probably add this for all servers in another PR.

Reply:
https://stellarorg.atlassian.net/browse/ANCHOR-669